clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -61,28 +61,35 @@ class ServerGroupController {
 
   @PreAuthorize("hasPermission(#application, 'APPLICATION', 'READ') and hasPermission(#account, 'ACCOUNT', 'READ')")
   @RequestMapping(value = "/applications/{application}/serverGroups/{account}/{region}/{name:.+}", method = RequestMethod.GET)
-  ServerGroup getServerGroupByApplication(@PathVariable String application, // needed for @PreAuthorize
+  ServerGroup getServerGroupByApplication(@RequestParam(required = false, value = 'expand', defaultValue = 'true') String expand,
```

Comment:
Can you put the `@RequestParam` after the `PathVariable` -- I believe that'll be more consistent with where we're doing this elsewhere.

We're calling it `expand` on the request parameter but `includeDetails` on the underlying provider interfaces -- is there a reason to not keep the naming consistent?